webClient/package.json

```diff
@@ -36,7 +36,7 @@
     "@angular/platform-browser-dynamic": "~6.0.9",
     "@angular/router": "~6.0.9",
     "@types/node": "^6.0.112",
-    "@zlux/file-explorer": "git+ssh://git@github.com:zowe/zlux-file-explorer.git#staging",
+    "@zowe/zlux-angular-file-tree": "0.0.3-pr-72-24-20200519152533",
```

Comment:
this PR to try end-to-end after zowe artifactory publishing of zlux-file-explorer component

Reply:
npm config set registry https://zowe.jfrog.io/zowe/api/npm/npm-release/